package.json

```diff
@@ -21,6 +21,7 @@
     "@angular/common": "^5.0.1",
     "@angular/compiler": "^5.0.1",
     "@angular/core": "^5.0.1",
+    "@angular/elements": "github:angular/elements-builds#labs/elements",
```

Comment:
There is no published NPM for this?

Reply:
That's right, it's still an experimental library.